test-fixture.html

```diff
@@ -313,9 +315,20 @@
     });
 
   try {
-    document.registerElement('test-fixture', {
-      prototype: TestFixturePrototype
-    });
+    if (window.customElements) {
+      function TestFixture() {
+        if (window.Reflect) {
+          return Reflect.construct(HTMLElement, [], TestFixture)
```

Comment:
Oh, weird, I didn't realize `new.target` and `Reflect` weren't more closely linked. Looks like you're right though; the kangax compat table claims that a lot of polyfills only do `Reflect`:
![screen shot 2016-08-24 at 16 54 02](https://cloud.githubusercontent.com/assets/406614/17952004/6257c6c6-6a1b-11e6-9e33-94a590223144.png)